AlphaWallet/Core/NFT/Enjin/EnjinNetworkProvider.swift

```diff
@@ -24,15 +23,11 @@ final class EnjinNetworkProvider {
         return ApolloClient(networkTransport: transport, store: EnjinNetworkProvider.store)
     }()
 
-    init(queue: DispatchQueue) {
-        self.queue = queue
-    }
-
     typealias GetEnjinBalancesResponse = (balances: Enjin.MappedEnjinBalances, owner: AlphaWallet.Address)
 
     func getEnjinBalances(forOwner owner: AlphaWallet.Address, offset: Int, sum: Enjin.MappedEnjinBalances = [:], limit: Int = 50, completion: @escaping (Swift.Result<GetEnjinBalancesResponse, EnjinError>) -> Void) {
 
-        graphqlClient.fetch(query: GetEnjinBalancesQuery(ethAddress: owner.eip55String, page: offset, limit: limit), queue: queue) { response in
+        graphqlClient.fetch(query: GetEnjinBalancesQuery(ethAddress: owner.eip55String, page: offset, limit: limit), queue: .global()) { response in
```

Comment:
I look down the call hierarchy and eventually, the queue is the `callbackQueue` argument, implying it is used only for invoking the `completionHandler`.

```
func send<Operation: GraphQLOperation>(operation: Operation,
                                         cachePolicy: CachePolicy,
                                         contextIdentifier: UUID?,
                                         callbackQueue: DispatchQueue,
                                         completionHandler: @escaping (Result<GraphQLResult<Operation.Data>, Error>) -> Void) -> Cancellable
```

So we should use `.main` and not `.global()`. But since the arg defaults to `.main`, we can leave it out completely to become:

```
graphqlClient.fetch(query: GetEnjinBalancesQuery(ethAddress: owner.eip55String, page: offset, limit: limit)) { response in
```

Unless you think this is intensive (I don't know, you are more familiar with this), then `.global()` or the same custom queue mentioned for the other commit.